spinner_test.go

```diff
@@ -178,6 +178,23 @@ func TestBackspace(t *testing.T) {
 	s.Stop()
 }
 
+// TestColorError tests that if an invalid color string is passed to the Color
+// function, the invalid color error is returned
+func TestColorError(t *testing.T) {
+	s := New(CharSets[0], 100*time.Millisecond)
+
+	invalidColorName := "bluez"
+	validColorName := "green"
+
+	if s.Color(invalidColorName) != errInvalidColor {
```

Comment:
I don't like this, I should be checking for _nil_ here, not simply the fact that it is not a specific error.